lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -553,16 +549,16 @@ def self.from_manifest_hash(manifest_hash)
         trigger['minutes_duration'] = Integer(manifest_hash['minutes_duration'])
       end
 
-      if start_date = manifest_hash['start_date']
-        start_date = Date.parse(start_date)
-        trigger['start_year']  = start_date.year
-        trigger['start_month'] = start_date.month
-        trigger['start_day']   = start_date.day
-      end
+      # manifests specify datetime in the local timezone, same as V1 trigger
+      # TODO: figure out how 0 0 0 manifests... OR... just check and pass along?
```

Comment:
What's being done here passes tests.. but any thoughts on the above question?

Reply:
Personally prefer to check and pass, not sure anyone _should_ be passing all zeroes.

Reply:
The V1 API allows 0s... if it wasn't clear, this is the method that takes an ITrigger and converts it to the equivalent V1 hash. When a `StartBoundary` comes in with the value `''` the question is ... how would that be stored?

It probably doesn't matter much in the long term as this conversion will go the way of the dodo once the other PR is merged.

Reply:
Oh, I thought this was converting from the hash to the ITrigger.  The comment lines _seem_ to live inside of https://github.com/puppetlabs/puppetlabs-scheduled_task/pull/33/files#diff-ede2463e358119c5b73d7d80617b78d2L514

Reply:
I actually just realized (after debugging some more to resolve this point) that I completely screwed something up - values for `start_year`, `start_month` and `start_day` are set in the hash for the tests, but we're looking at `start_date` only now because I misinterpreted the earlier code.

The original code allowed *both* a `start_date` value and individual values for the constituent pieces, allowing `start_date` (when specified) to take precedence (which is how things are written in a manifest).  I don't really understand why we allow this ambiguity, but we should probably file a ticket to take a quick audit (probably someone in the past got confused about manifest hash vs V1 esque hash).

Fixing to follow old behavior for now, but I think we have a few additional things to do later.

Reply:
Strong preference for date over the weird year/month/day individual specification, especially for humans.